docs/embroider-css-modules/ember-cli-build.js

```diff
@@ -39,7 +39,7 @@ module.exports = function (defaults) {
             ? '[sha512:hash:base64:5]'
             : '[path][name]__[local]',
           mode: (resourcePath) => {
-            const hostAppWorkspaceDir = `${options.workspaceDir}/${app.name}`;
+            const hostAppWorkspaceDir = options.workspaceDir;
```

Comment:
**bugfix**

By making the package name (app name) different from the app's root folder name, I found a bug in the current implementation for `hostAppWorkspaceDir`.

For example, when the package name is `docs-app-for-embroider-css-modules` but the root folder name `docs-app`, we will get the following values:

```sh
# hostAppWorkspaceDir
/private/var/folders/bg/<some hash>/T/embroider/4620d5/docs-app-for-embroider-css-modules

# resourcePath
/private/var/folders/bg/<some hash>/T/embroider/4620d5/docs-app/controllers/products.css
```

As a result, `isHostAppPath` will always equal to `false` (i.e. locally defined styles will never take place).